### PR TITLE
fix: otel ports

### PR DESCRIPTION
**Pull request by rahulk789**
## Describe your changes
change default otel ports from 4317 and 4318 to 7777 and 7778
## Checklist before requesting a review
- [ ] Have I performed a self-review?
- [ ] Have I added regression or unit tests (where it makes sense) for my changes?
- [ ] Have I updated the README if changes have resulted in it being out of date?
- [ ] Have I checked to ensure my PR only introduces the changes it's intended to? E.g. No extraneous formatting changes.
- [ ] Have I ensured that there are no performance regressions by checking benchmark results?
- [ ] Is this a breaking change? If yes, please describe areas affected and request reviews from developer stakeholders.

### CODE REVIEW

File: cmd/daemon.go [hunk 1]
```diff
@@ -55,13 +55,13 @@ var startDaemonCmd = &cobra.Command{
...
 		if otel_enabled {
-			_, err := utils.InitOtel(cmd.Context(), rootCmd.Version)
+			_, err := utils.InitOtel(ctx, rootCmd.Version)
 			if err != nil {
 				log.Warn().Err(err).Msg("Failed to initialize otel")
 				return err
 			}
-			log.Debug().Msg("otel initialized")
-
+			log.Info().Msg("initializing standard otel tracer")
```
Comment (swarnimarun) [inline cmd/daemon.go:63]:
```suggestion
			log.Info().Msg("initialized standard otel tracer")
```

File: pkg/utils/otel.go [hunk 2]
```diff
@@ -18,6 +19,7 @@ import (
 // setupOTelSDK bootstraps the OpenTelemetry pipeline.
 // If it does not return an error, make sure to call shutdown for proper cleanup.
 func InitOtel(ctx context.Context, version string) (shutdown func(context.Context) error, err error) {
+	log.Info().Msg("initializing standard otel tracer")
 	telemetryOn := viper.GetBool("otel_enabled")
```
Comment (swarnimarun) [inline pkg/utils/otel.go:23]:
```suggestion
```

File: pkg/utils/otel.go [hunk 3]
```diff
@@ -62,8 +68,12 @@ func newPropagator() propagation.TextMapPropagator {
 }
 
 func newTraceProvider(ctx context.Context, version string) (*trace.TracerProvider, error) {
+
+	grpcPort := viper.GetInt("otel_port")
+	address := fmt.Sprintf("127.0.0.1:%d", grpcPort)
```
Comment (swarnimarun) [inline pkg/utils/otel.go:73]:
```suggestion
	otelGrpcPort := viper.GetInt("otel_port")
	address := fmt.Sprintf("127.0.0.1:%d", otelGrpcPort)
```
nit, we have a port for grpc service as well

**Review by swarnimarun — COMMENTED** [review]
lgtm

File: pkg/utils/otel.go (hunk 3 above)
Comment (rahulk789) [reply pkg/utils/otel.go:73]: changing

**Review by nravic — APPROVED** [review]
lgtm, great work!

**Review by nravic — APPROVED** [review]
`./build-start-daemon.sh --systemctl --no-build --otel --k8s` in setup-host still has the --otel flag. So even with noop removed, the tracer starts in k8s

**Review by nravic — CHANGES_REQUESTED** [review]
added some comments

**Review by bsmithai — APPROVED** [review]
lgtm

**Pull request merged**